### PR TITLE
Rename references to function parameters inside function bodies

### PR DESCRIPTION
**Pull request by ahoppen**
And some trivial clean-up that can be reviewed commit-by-commit.

rdar://122031097

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

File: Sources/SKSupport/LineTable.swift [hunk 1]
```diff
@@ -141,6 +141,20 @@ extension LineTable {
     return content.utf16.index(lineSlice.startIndex, offsetBy: utf16Column, limitedBy: lineSlice.endIndex)
   }
 
+  /// Returns `String.Index` of given logical position.
+  ///
+  /// - parameter line: Line number (zero-based).
+  /// - parameter utf8Column: UTF-8 column offset (zero-based).
```
Comment (bnbarham) [inline Sources/SKSupport/LineTable.swift:147]: Zero based line/column is super weird. Do we actually need it to be?

Comment (ahoppen) [reply Sources/SKSupport/LineTable.swift:147]: All of LSP is zero-based, so we tend to default to that as well.

**Pull request merged**